### DISCUSSION TITLE
UDP Connect

### DISCUSSION THREAD

**Discussion by im22361** [openthread/openthread#10363] [General]
I am using an ESP32 Thread border with an ESP32-C6 I have been able to use the  udp connect command from the 
 end device(ESP32-C6) to send a message to my laptop connected to the same wifi network as the border router. However when I try the same command from the border router I get the InvalidState.

**Comment by chshu** [comment]
@im22361 Could you please share more details to reproduce the issue? The SDK version, commands and the full logs.

**Reply by im22361** [reply]
Thank you, that issue is resolved, however I have another question.
I am able to send messages to my laptop via udp connect command in CLI.
It works, but I get this error in the command line: E (306541) OPENTHREAD: Failed to Send UDP message, err: -12
What is going wrong?
